prio/callbk.c

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2018, Henry Corrigan-Gibbs
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include "callbk.h"
+#include <nss.h>
+
+void nss_callbk_init(nss_struct *nss_pointer){
```

Comment:
Don't include references to NSS in the prio library. Move this code into pclient/main.c and ptest/mutest.c. It should be set only by users of libprio